gpt_index/vector_stores/qdrant.py

```diff
@@ -42,19 +37,8 @@ class GPTQdrantIndex(BaseGPTVectorStoreIndex[QdrantIndexStruct]):
         collection_name: (Optional[str]): name of the Qdrant collection
     """
 
-    index_struct_cls = QdrantIndexStruct
-
     def __init__(
-        self,
-        documents: Optional[Sequence[DOCUMENTS_INPUT]] = None,
-        index_struct: Optional[QdrantIndexStruct] = None,
-        text_qa_template: Optional[QuestionAnswerPrompt] = None,
-        llm_predictor: Optional[LLMPredictor] = None,
-        embed_model: Optional[BaseEmbedding] = None,
-        text_splitter: Optional[TextSplitter] = None,
-        client: Optional[Any] = None,
-        collection_name: Optional[str] = None,
-        **kwargs: Any
+        self, collection_name: str, client: Optional[Any] = None, **kwargs: Any
     ) -> None:
         """Init params."""
         import_err_msg = (
```

Comment:
Why do we do import check in constructor?  Could we move to module level?

Reply:
lazy import is so it doesn't need to be a package dependency

Reply:
Why not lazy import at module level instead of function level?